File: WikiMiner - Python/readFold.py
from numpy import *


def readFold(idFold, typeSet, out_path):
    filename = out_path + "codes-to-create-school/virtual_folds/fixed/" + typeSet + '_' + str(idFold) + '.txt'

    fileId = open(filename, 'r')
    In = loadtxt(fileId, delimiter=" ", dtype='int')
    fileId.close()

    foldIds = In

    return foldIds.reshape(-1,1)
# end

File: utils/matlab-codes-revised/readFold.m
% Codes by: Raoni Ferreira.

function [foldIds] = readFold(path_output, idFold,typeSet)

filename = strcat(path_output, 'codes-to-create-school\virtual_folds\fixed\',typeSet,'_',num2str(idFold),'.txt');

fileId = fopen(filename,'r');
in = textscan(fileId,'%d','Delimiter','');
fclose(fileId);

foldIds = in{1};
end
